app/redux/ducks/interventions.js

```diff
@@ -34,6 +38,9 @@ export default function reducer(state = initialState, action = {}) {
       sugarClient.unsubscribeFrom(action.payload);
       return state;
     default:
+      if (process.env.BABEL_ENV !== 'test') {
+        console.log(`Uknown intervention action: ${action.payload}`);
```

Comment:
The Redux dev tools will log this for you, so you're better off installing those in the browser than adding console logs.